charts/guac/values.yaml

```diff
@@ -190,10 +200,26 @@ guac:
     ingest: false
     jobName: ingest-guac-data
 
+  ingress:
+    enabled: false
+    # ingressClassName:
+    # webuiHostname:
+    # apiHostname:
+
+  backend:
+    ent:
+      db-driver: postgres
+      db-address: postgres://guac:guac@host:port/dbName?sslmode=disable
+      db-migrate: true
+      db-debug: false
+
+  pubSubAddr: awssqs://sqs.[aws_region].amazonaws.com/[account_id]/[queue_name]?region=[aws_region]
```

Comment:
Also, this should wait until a new release is done for guac. As the current release does not support the pubsub or blob store.